docs/stock-analysis-plan.md

```
@@ -0,0 +1,1036 @@
+# Stock Analysis Extension Plan (v3)
+
+## Overview
+
+Add a `!sa SYMBOL` command that provides AI-powered stock analysis by combining:
+
+1. **Finnhub** — real-time quote + company profile (existing)
+2. **Exa** — web search for recent news and financial journalism
+3. **Gemini** — synthesizes Finnhub data + Exa highlights into a concise analysis
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)